rigs/heft-web-rig/profiles/library/config/typescript.json

```diff
@@ -31,12 +31,12 @@
   ]
 
   /**
-   * Specifies the intermediary folder that Jest will use for its input.  Because Jest uses the
+   * Specifies the intermediary folder that tests will use.  Because Jest uses the
    * Node.js runtime to execute tests, the module format must be CommonJS.
    *
    * The default value is "lib".
    */
-  // "emitFolderNameForJest": "lib-commonjs",
+  // "emitFolderNameForTests": "lib-commonjs",
```

Comment:
@iclanton Seems to be that a web library should transpile to ES5, which means it needs to a secondary emit into `"lib-commonjs"` for Jest, right?